pygramed/__init__.py

```diff
@@ -72,6 +74,16 @@ def __init__(self) -> None:
         self.url = f'{self._base_url}/products/'
 
     def retrieve(self,
+                 category: Optional[Union[List[str], str]] = None,
+                 limit: Optional[int] = None) -> List[Product]:
+        return self._retrieve(category, limit, True)
+
+    def retrieve_async(self,
```

Comment:
meh.. This would mean to have sync/async objects on `PyGramedia` which would do pretty much same logic but only one returns iterated loop.

I think having methods `retrieve` and `retrieve_async` is enough. Maybe I could move `self._loop.run_until_complete(coroutine)` from `_retrieve` to `retrieve` so it would be more obvious that only iteration of that loop changed?

Reply:
Good then, will merge your PR after `datetime` format finished.
Thank you! 🎉